editor/src/components/canvas/commands/show-grid-controls-command.ts

```diff
@@ -8,20 +8,23 @@ export interface ShowGridControlsCommand extends BaseCommand {
   target: ElementPath
   targetCell: GridCellCoordinates | null
   rootCell: GridCellCoordinates | null
+  rootCellIsValidTarget: boolean | null
 }
 
 export function showGridControls(
   whenToRun: WhenToRun,
   target: ElementPath,
   targetCell: GridCellCoordinates | null,
   rootCell: GridCellCoordinates | null,
+  rootCellIsValidTarget: boolean | null,
```

Comment:
very minor, but a string literal union is much better for these flags

Reply:
[`021fdcb` (#6606)](https://github.com/concrete-utopia/utopia/pull/6606/commits/021fdcb807c4b6e62940f2a690e0674e6fb2a91f)